circus-train-core/src/test/java/com/hotels/bdp/circustrain/core/DiffGeneratedPartitionPredicateTest.java

```diff
@@ -21,10 +22,10 @@
 import static org.mockito.Matchers.anyString;
 import static org.mockito.Mockito.when;
 
-import static com.hotels.bdp.circustrain.core.metastore.HiveEntityFactory.newFieldSchema;
-import static com.hotels.bdp.circustrain.core.metastore.HiveEntityFactory.newPartition;
-import static com.hotels.bdp.circustrain.core.metastore.HiveEntityFactory.newStorageDescriptor;
-import static com.hotels.bdp.circustrain.core.metastore.HiveEntityFactory.newTable;
+import static com.hotels.bdp.circustrain.core.HiveEntityFactory.newFieldSchema;
```

Comment:
The metastore package should probably be re-introduced as we don't want lots of classes at the root of "core"